Source/SwiftLintBuiltInRules/Rules/Idiomatic/UnneededBreakInSwitchRule.swift

```diff
@@ -65,3 +82,48 @@ private final class UnneededBreakInSwitchRuleVisitor: ViolationsSyntaxVisitor {
         violations.append(statement.item.positionAfterSkippingLeadingTrivia)
     }
 }
+
+private final class UnneededBreakInSwitchRewriter: SyntaxRewriter, ViolationsSyntaxRewriter {
+    private(set) var correctionPositions: [SwiftSyntax.AbsolutePosition] = []
+    let locationConverter: SourceLocationConverter
+    let disabledRegions: [SourceRange]
+
+    init(locationConverter: SourceLocationConverter, disabledRegions: [SourceRange]) {
+        self.locationConverter = locationConverter
+        self.disabledRegions = disabledRegions
+    }
+
+    override func visit(_ node: SwitchCaseSyntax) -> SwitchCaseSyntax {
+        guard node.statements.count > 1,
+              let statement = node.statements.last,
+              let breakStatement = statement.item.as(BreakStmtSyntax.self),
+              breakStatement.label == nil,
```

Comment:
This can be extracted into an extension method on `SwitchCaseSyntax` to have the conditions for a violation only implemented once.

Reply:
Fixed in 004b6f4